docs/en/stack/ml/df-analytics/dfa-regression.asciidoc

```diff
@@ -88,7 +85,47 @@ gradient boosting methodologies.
 //end::regression-algorithms[]
 
 
-[discrete]
+[[dfa-regression-lossfunction]]
+===== Loss functions for {regression} analyses
+
+Loss function measures how well a given {ml} model fits the specific 
+data set. The loss function boils down all the different under- and 
```

Comment:
Since the previous sentence started with the same phrase, I think you can shorten this one:
```suggestion
data set. It boils down all the different under- and 
```